.github/workflows/ci.yml

```diff
@@ -142,11 +142,6 @@ jobs:
           curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
           sudo apt-get -y install nodejs
       
-      - name: run `reqwest-wrap` WASM tests
```

Comment:
I had these tests here originally to test the timeout PR that was applied on top of `reqwest`. After moving it under `ergo-rest`, the WASM tests for it no longer compiles, due to a really weird error in the dependency hierarchy (for reference it's exactly this error message: https://github.com/alexcrichton/wait-timeout/issues/25).

I've opted to delete it for now.